pallets/rc-migrator/src/lib.rs

```diff
@@ -122,9 +122,18 @@ pub enum PalletEventName {
 #[derive(Encode, Decode, Clone, Default, RuntimeDebug, TypeInfo, MaxEncodedLen, PartialEq, Eq)]
 pub enum MigrationStage<AccountId, BlockNumber, BagsListScore, AccountIndex, VotingClass, AssetKind>
 {
-	/// The migration has not yet started but will start in the next block.
+	/// The migration has not yet started but will start in the future.
 	#[default]
 	Pending,
```

Comment:
make sure to change code that was relying on old meaning of `Pending` (at the very least `self.is_ongoing()`)

Reply:
updated